File: src/main/java/de/tectoast/emolga/utils/sql/managers/ReplayCheckManager.java
package de.tectoast.emolga.utils.sql.managers;

import de.tectoast.emolga.utils.sql.base.DataManager;
import de.tectoast.emolga.utils.sql.base.columns.LongColumn;

public class ReplayCheckManager extends DataManager {

    final LongColumn CHANNELID = new LongColumn("channelid", this);
    final LongColumn MESSAGEID = new LongColumn("messageid", this);

    public ReplayCheckManager() {
        super("replaycheck");
        setColumns(CHANNELID, MESSAGEID);
    }

    public void set(long tcid, long mid) {
        insertOrUpdate(CHANNELID, tcid, tcid, mid);
    }
}

File: src/main/kotlin/de/tectoast/emolga/utils/sql/managers/ReplayCheckManager.kt
package de.tectoast.emolga.utils.sql.managers

import de.tectoast.emolga.utils.sql.base.DataManager
import de.tectoast.emolga.utils.sql.base.columns.LongColumn

object ReplayCheckManager : DataManager("replaycheck") {
    private val CHANNELID = LongColumn("channelid", this)
    private val MESSAGEID = LongColumn("messageid", this)

    init {
        setColumns(CHANNELID, MESSAGEID)
    }

    operator fun set(tcid: Long, mid: Long) {
        replaceIfExists(tcid, mid)
    }
}